src/cgroups/v1/blkio.rs

```diff
@@ -106,28 +106,44 @@ impl StatsProvider for Blkio {
 
 impl Blkio {
     fn apply(root_path: &Path, blkio: &LinuxBlockIo) -> Result<()> {
-        for trbd in &blkio.blkio_throttle_read_bps_device {
+        for trbd in blkio
+            .throttle_read_bps_device
+            .as_ref()
+            .context("no throttle_read_bps_device in linux blkio")?
```

Comment:
This is something that should be resolved separately from this PR, but I think there needs to be an implementation for when the config is missing.
I will compile a list of missing implementations of this kind and put them in an issue at a later date.

Reply:
https://github.com/containers/youki/issues/184